common/djangoapps/enrollment/api.py

```diff
@@ -270,30 +292,45 @@ def get_course_enrollment_details(course_id):
         }
 
     """
-    cache_key = u"enrollment.course.details.{course_id}".format(course_id=course_id)
+    
+    cache_key = \
+        u'enrollment.course.details.{course_id}.{include_expired}'.format(course_id=course_id,
+                                                                         include_expired=include_expired)
```

Comment:
Please make sure you're following the style guidelines for breaking up long lines.
```
cache_key =  u'enrollment.course.details.{course_id}.{include_expired}'.format(
     course_id=course_id,
     include_expired=include_expired
)
```